deltachat-ios/Controller/ConnectivityViewController.swift

```diff
@@ -85,6 +114,18 @@ class ConnectivityViewController: WebViewViewController {
                 .appending(String.localized("connectivity_not_connected"))
         }
 
+        if isLowDataMode {
+            return "<span class=\"red dot\"></span>"
```

Comment:
i'd say, this is no error, it is disabled as for the other reasons.

```suggestion
            return "<span class=\"disabled dot\"></span>"
```